js/app/historyStore.js

```diff
@@ -49,6 +49,13 @@ const HistoryStore = new Lang.Class({
         'current-set': GObject.ParamSpec.object('current-set', 'current-set', 'current-set',
             GObject.ParamFlags.READABLE, SetObjectModel.SetObjectModel),
         /**
+         * Property: current-subset
+         *
+         * The model for the current subset in the history store.
+         */
+        'current-subset': GObject.ParamSpec.object('current-subset', 'current-subset', 'current-subset',
```

Comment:
> maybe it would be better to add a new `current-toplevel-set` property

That's what I had originally but moved to this because with `current-toplevel-set` I have to make more changes across other modules. `current-set` then has different semantics across history stores because in the case if course history store, when you click set on home page you _dont_ want to change `current-set`, but in other history stores you do